File: Tests/NyxCoreTests/Fixtures/export/06-get-with-urlencode.js
const params = new URLSearchParams();
params.append("q", "hello world");
params.append("lang", "en");

fetch("https://api.example.com/search" + "?" + params.toString(), {
  method: "GET",
  headers: {
    "Accept": "application/json",
  },
})
  .then((response) => response.json())
  .then((data) => console.log(data))
  .catch((error) => console.error(error));
// not translated: -s, -S

File: Tests/NyxCoreTests/Fixtures/export/06-get-with-urlencode.py
import requests

response = requests.request(
    "GET",
    "https://api.example.com/search",
    params={
        "q": "hello world",
        "lang": "en",
    },
    headers={
        "Accept": "application/json",
    },
)

print(response.status_code, response.text)
# not translated: -s, -S
